api-reference/10 UI Widgets/dxForm/1 Configuration/colCount.md

```diff
@@ -47,6 +47,43 @@ For extra small screens, this option always equals *1* to make the widget adapti
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxForm ...>
+            <DxColCountByScreen :xs="2"/>
+        </DxForm>
+    </template>
+    <script>
+    import DxForm, { DxColCountByScreen } from 'devextreme-vue/form';
+
+    export default {
+        components: {
+            DxForm,
+            DxColCountByScreen
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import Form, { ColCountByScreen } from 'devextreme-react/form';
+
```

Comment:
Since the release of DevExtreme v20.2 we write React snippets in functional style. See an example below.